model/marshal.go

```diff
@@ -670,3 +670,35 @@ func writeHex(w *fastjson.Writer, v []byte) {
 		w.RawByte(hextable[v&0x0f])
 	}
 }
+
+// MarshalFastJSON writes the JSON representation of v to w.
+func (v *Metric) MarshalFastJSON(w *fastjson.Writer) error {
+	w.RawByte('{')
+	w.RawString("\"type\":")
+	w.String(v.Type)
+	if len(v.Counts) > 0 {
+		w.RawString(",\"buckets\":")
```

Comment:
apm-server expects it to be called "values": https://github.com/elastic/apm-server/blob/217b3841177a1ac3dc068a8fa982c871427a2273/docs/spec/v2/metricset.json#L49

I think it might be prudent to rename all "buckets" to "values" to keep aligned

Reply:
ah oops, thanks